extensions/adobe/experience/customerJourneyManagement/ajo-entity-mixins.schema.json

```diff
@@ -5,131 +5,178 @@
     "you may not use this file except in compliance with the License. You may obtain a copy",
     "of the License at https://creativecommons.org/licenses/by/4.0/"
   ],
-  "$id": "https://ns.adobe.com/experience/customerJourneyManagement/ajo",
+  "$id": "https://ns.adobe.com/xdm/ajo",
   "$schema": "http://json-schema.org/draft-06/schema#",
-  "title": "Adobe Profile - AJO Entities",
-  "description": "Denormalized AJO Entities.",
+  "title": "AJO Entity Fields",
+  "description": "A set of meta-data fields related to various AJO entities which are related to message sent to profile.",
   "type": "object",
   "meta:extensible": true,
   "meta:abstract": false,
-  "meta:extends": ["https://ns.adobe.com/xdm/data/record"],
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/ajo/entity"],
```

Comment:
After you have changed the $id of the class, replace this with the new class ID
"https://ns.adobe.com/experience/customerJourneyManagement/ajoEntity"